pkg/model/validation/validation.go

```diff
@@ -14,28 +14,34 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package resource
+package validation
```

Comment:
From a design point of view I don't like this package. `pkg/model/validation`? Is the validation a model?

I see what you did removing the duplicated code in `pkg/model/resource/validation.go` and `pkg/plugin/internal/validations.go` but maybe we can think of a better path. A `validation` package makes sense, but being it a subpackage of `cli`, `model``, `plugin` or `scaffold` doesn't seem right. My suggestion would be to move it to `pkg/internal/validation/dns.go` as they are all DNS related validations for now. That way we would not have to mantain backwards compatibility of those functions (that are exported) but will still be accessible from almost everywhere. (NOTE: moving `internal/*` to `pkg/internal/*` looks like a nice thing to do if we end up removing the dependencies from `cmd`.

Reply:
I agree with @Adirio here as well. 👍